frontend/apps/web/components/jobs/SchemaTable/SchemaTableToolBar.tsx

```diff
@@ -186,7 +186,7 @@ export function SchemaTableToolbar<TData>({
                   const identityType =
                     constraintHandler.getIdentityType(colkey);
                   const newJm =
-                    isGenerated || identityType
+                    isGenerated && !identityType
```

Comment:
What about serial? Can we set everything to passthrough?

Reply:
I'm not sure serial data types are reflected in these booleans, are they?

We can set serials to passthrough though, no?